kafka-quarkus-dev/README.adoc

```diff
@@ -66,7 +66,7 @@ For this quick start, you'll use the Quarkus sample code from the {product} {sam
 ----
 $ mkdir guides && git clone {samples-git-repo}
```

Comment:
@wtrocki can you fix this line, it doesn't clone the repo in the guides directory but it just creates an empty guides dir and then check out the repo with the repo name, the right command is:
```
git clone {samples-git-repo} guides
```

Reply:
```suggestion
git clone {samples-git-repo} guides
```